### PR TITLE
release(v1.1.1): prepare release

### PR DESCRIPTION
**Pull request by smira**
This is the official v1.1.1 release.

Signed-off-by: Andrey Smirnov <andrey.smirnov@talos-systems.com>

### CODE REVIEW

File: website/content/v1.1/reference/cli.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ talosctl cluster create [flags]
       --bad-rtc                                  launch VM with bad RTC state (QEMU only)
       --cidr string                              CIDR of the cluster network (IPv4, ULA network for IPv6 is derived in automated way) (default "10.5.0.0/24")
       --cni-bin-path strings                     search path for CNI binaries (VM only) (default [/home/user/.talos/cni/bin])
-      --cni-bundle-url string                    URL to download CNI bundle from (VM only) (default "https://github.com/siderolabs/talos/releases/download/v1.1.0/talosctl-cni-bundle-${ARCH}.tar.gz")
+      --cni-bundle-url string                    URL to download CNI bundle from (VM only) (default "https://github.com/siderolabs/talos/releases/download/v1.1.1-dirty/talosctl-cni-bundle-${ARCH}.tar.gz")
```
Comment (frezbo) [inline website/content/v1.1/reference/cli.md:96]:
```suggestion
      --cni-bundle-url string                    URL to download CNI bundle from (VM only) (default "https://github.com/siderolabs/talos/releases/download/v1.1.1/talosctl-cni-bundle-${ARCH}.tar.gz")
```

**Review by frezbo — CHANGES_REQUESTED** [review]
the `-dirty`

File: website/content/v1.1/reference/cli.md (hunk 1 above)
Comment (smira) [reply website/content/v1.1/reference/cli.md:96]: yes, something went wrong ;)

**Review by frezbo — APPROVED** [review]

**Pull request merged**